tests/datasource/fluent/test_sqlite_datasource.py

```diff
@@ -59,12 +58,16 @@ def test_connection_string_that_does_not_start_with_sqlite():
             name=name,
             connection_string=connection_string,
         )
-    assert str(e.value) == (
-        "1 validation error for SqliteDatasource\n"
-        "connection_string\n"
-        "  URL scheme not permitted (type=value_error.url.scheme; "
-        f"allowed_schemes={SqliteDsn.allowed_schemes})"
-    )
+    # the first error is due to missing a config template string
+    assert e.value.errors()[1]["msg"] == "URL scheme not permitted"
+    assert e.value.errors()[1].get("ctx") == {
```

Comment:
I'm guessing these are from sqlalchemy? We don't really have control over them and would be ok with dropping this assert or just checking `sqlite` is present.

Reply:
No this is the custom pydantic `SQLiteDSN` type and related validation error @NathanFarmer created.